packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/add/dedicatedCloud-datacenter-add.controller.js

```diff
@@ -96,7 +97,7 @@ export default class {
 
   convertPriceValueToDisplay(value) {
     const priceAsString = new Intl.NumberFormat(
-      this.ovhSubsidiary.toLowerCase(),
+      this.coreConfig.getUserLocale().replace('_', '-'),
```

Comment:
**nitpick:** The price should always be based on Subsidiary. If a French user, switches to Italian language. the price displayed will not be consistent.